topk-js/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+edition = "2021"
+name = "topk-js"
+description = "TopK JavaScript SDK with TypeScript support"
+version = "0.0.0"
```

Comment:
note: this will need to follow the same versioning scheme as `topk-py` (and `topk-rs`) since they all share the same code